demo/src/App.vue

```diff
@@ -197,11 +198,9 @@ export default {
       m.data.text = message.data.text;
     },
     removeMessage(message){
-      if (confirm('Delete?')){
```

Comment:
@arabakevin @zedtux In the review that I did earlier, I failed to consider this PR from a high-level view. I realized that this diff basically moves this single line above _inside_ the package.

The benefit is that a user will not have to write this condition in the `remove` handler.

Let's look at the cons:

- The simple condition is replaced by two props: `showConfirmationDeletion` and `confirmationDeletionMessage`. I'd say it's a double whammy of increased complexity: on the user side where they have to manage the state and pass the props, and on this package's side with a larger public API and internal state. Essentially, this one line here versus this whole diff _plus_ the diff for adding `confirmationDeletionMessage` prop (which isn't here).
- The logic for performing a removal is now split between the template and the remove event handler (and possibly `data` if the above props are initially defined there). In terms of readability, I think having any confirmation logic _inside_ the handler is much better and keeps things contained.
- I'd say `confirm` here was used just for demo purposes. Browser default dialog boxes should not actually be used for something like this as they provide poor UX (preventing the user from interactive with anything else on the page, and blocking the main JS thread).

For these reasons, I'm proposing to revert this changeset. A user can just provide their own logic for confirming deletion if they must, which I think is a flexible and appropriate approach for this.

The larger idea is to keep this package light-weight in terms of its API and also bundle size so that it loads fast, while remaining maintainable over long term.

cc @mattmezza 

@arabakevin @zedtux If the above sounds okay, please move the logic for handling confirmations back to the remove event handler and discontinue use of the props. Else, please let me know if I missed anything above.